### ISSUE TITLE
Develop Plotly data viewer

### ISSUE THREAD

**Issue by astrofrog** [glue-viz/glue#1967]
Now that Plotly includes the ability to export 'static' HTML pages we could consider making a data viewer than is based on the HTML Plotly output, using QWebEngine to show it (like we do for WWT)

**Comment by DumbMachine** [comment]
Hey, I would like to work on this. I have prior experience with Plotly. If this issue is still up for grabs, could you tell more about it?

**Comment by astrofrog** [comment]
@DumbMachine - great! We are now developing plot.ly-specific functionality in https://github.com/glue-viz/glue-plotly rather than in this repository. A good place to start would be to work on https://github.com/glue-viz/glue-plotly/issues/1 as once that is done some of the pieces can probably be re-used to implement https://github.com/glue-viz/glue-plotly/issues/2 or https://github.com/glue-viz/glue-plotly/issues/3. I've tried to provide more details in https://github.com/glue-viz/glue-plotly/issues/1, which is a good first issue to work on.

**Comment by astrofrog** [comment]
Closing this in favor of https://github.com/glue-viz/glue-plotly/issues/2
